### DISCUSSION TITLE
How to actually get started with Svelte/SvelteKit ?

### DISCUSSION THREAD

**Discussion by Extarys** [sveltia/sveltia-cms#250] [Q&A] [answered]
The instructions aren't clear for me, and it seems no one is talking about sveltia-cms on the internet enough to gather knowledge on how to do things.

On one hand, it suggests to install/follow instructions for Decap, but decap doesn't have a template for svelte. Even if it did, it then imports the js file from a cdn, not locally.

Then there's the manual option that says 
> The [manual initialization](https://decapcms.org/docs/manual-initialization/) flow with the init method is the same as for Netlify/Decap CMS.

But the decap instructions still feel... incomplete ?

I then thought about if it would be possible to use sveltekit, so some articles could require authentication (hooks and tighter integration with a main website), but then the "manual initialization" wouldn't apply with sveltekit.

If someone have some time to guide me through how to actually set up sveltia, I would be very pleased :D it seems like a great piece of code :)

**Comment by kyoshino** [comment, marked as answer]
I have created Sveltia CMS mainly for my client SvelteKit sites, so SvelteKit support is guaranteed. You don’t need to use the manual initialization method. Just use the [normal installation method](https://decapcms.org/docs/install-decap-cms/) of `index.html` with `<script>`.

My quick search finds a [Decap + SvelteKit template](https://github.com/salty-muffin/sveltekit-decapcms), though I haven’t tried it yet.

You can also sneak a peek at one of my client sites on a [public repository](https://github.com/zero-in-on/zeroinon-web), though it’s not open source. The key part is [`import.meta.glob`](https://vite.dev/guide/features#glob-import), which is actually a feature of Vite, not SvelteKit, that allows multiple entries to be read at once like [this file](https://github.com/zero-in-on/zeroinon-web/blob/main/src/lib/services/marketing/news.js).

Later I’ll certainly provide a SvelteKit starter kit.
